crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -41,6 +41,19 @@ def roles_for_select(selected)
     )
   end
 
+  # Possible values for nodes default file system
+  def fs_for_select(selected)
+    options_for_select(
+      [
+        ["ext3", "ext3"],
```

Comment:
Well, as for now, we still use ext3 for RAID. But it's just an option.

ext4 is still default, so you user has to change it explicitely. My idea was that further explanation when to use btrfs and possibly why not should go to Deployment guide.

But if we would consider anything else than ext4 as unsupported, than hint in UI definitely makes sense